src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,6 +5,9 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private String carBody;
```

Comment:
According to the checklist, custom fields should not be exposed directly. Consider using private access modifiers and providing getters and setters for these fields.